src/components/scenes/Loans/LoanCreateScene.tsx

```diff
@@ -181,10 +181,11 @@ export const LoanCreateScene = (props: Props) => {
   // Convert required collateral in fiat -> required collateral in crypto
   const { assetToFiatRate: collateralToFiatRate } = useTokenDisplayData({
     tokenId: collateralTokenId,
-    wallet: borrowEngineWallet
+    wallet: srcWallet ?? borrowEngineWallet // borrowEngineWallet given just so hook doesn't puke
```

Comment:
I think replacing the usage of `useTokenDisplayData` with something that doesn't depend on a wallet would be more appropriate. You're only actually using it to grab an exchange rate so I think this would work:

const collateralToFiatRate = useSelector(state => state.exchangeRates[`${hardCollateralCurrencyCode}_${isoFiatCurrencyCode}`]) ?? '0'

Reply:
IsoFiatCurrencyCode does come from the wallet. 
We had duplication of this sort of thing all over the place and everyone was doing it differently - that was the drive of the initial refactor of all that into useTokenDisplayData. Even though it's just the exchange rate, the motivation of creating this hook is to stop doing exactly that in the scenes and components.

Reply:
I get the appeal of removing duplicate code (and ensuring it's always done the same)but the helper function isn't appropriate for the situation it's used where the wallet could be null. I think it's better to use safe defaults rather than useless replacement data which could negatively impact something further down the line. This would work and matches most of the srcWallet declarations above it:

const srcWalletFiat = srcWallet != null ? getWalletFiat(srcWallet).isoFiatCurrencyCode : ''
const collateralToFiatRate = useSelector(state => state.exchangeRates[${hardCollateralCurrencyCode}_${isoFiatCurrencyCode}]) ?? '0'

Any calculation further down would use a safer '0' instead of a bogus exchange rate returned by using the the borrowEngineWallet fiat.